rpcs3/Emu/Cell/Modules/cellSysutil.cpp

```diff
@@ -62,6 +64,23 @@ extern u32 sysutil_send_system_cmd(u64 status, u64 param)
 
 	if (auto cbm = g_fxo->try_get<sysutil_cb_manager>())
 	{
+		if (status == CELL_SYSUTIL_DRAWING_BEGIN)
+		{
+			if (cbm->draw_cb_started++)
```

Comment:
Might as well use try inc 1 here

Reply:
There can't normally but if it happens atleast END will be called lastly.